libs/core/tests/unit_tests/prompts/test_chat.py

```diff
@@ -533,3 +533,16 @@ def test_chat_prompt_message_placeholder_partial() -> None:
     assert prompt.format_messages() == []
     prompt = prompt.partial(history=[("system", "foo")])
     assert prompt.format_messages() == [SystemMessage(content="foo")]
+
+
+def test_messages_prompt_accepts_list() -> None:
+    prompt = ChatPromptTemplate.from_messages([MessagesPlaceholder("history")])
+    value = prompt.invoke([("user", "Hi there")])  # type: ignore
```

Comment:
his is confusing -- are we sure we want to support?, why is it a list of 2-tuples rather than only a 2-tuple?

Reply:
We already support `prompt.invoke({"history": [("user", "hi there"]})`